collections/_authors/OpenINFBot.md

```diff
@@ -0,0 +1,7 @@
+---
+permalink: /OpenINFBot
+redirect_from: /openinfbot
+redirect_to: https://x.com/OpenINFBot
```

Comment:
```suggestion
redirect_to: https://github.com/OpenINFBot
```

i would like to add an url for the new OpenINFBot account on X, so let's link to gh profile